src/style.css

```diff
@@ -0,0 +1,20 @@
+html {
```

Comment:
```suggestion
body {
```

Better to apply font-family in 'body' selector